test/languages_test.js

```diff
@@ -56,3 +46,12 @@ tape.test('verify language files structure', function(assert) {
 
     assert.end();
 });
+
+tape.test('verify that instructions are only returned for user requested languages', function(assert) {
+    var translations = languages.get(['en', 'fr']);
+
+    assert.deepEqual(Object.keys(translations).sort(), ['fr', 'en'].sort(),
+    'only returns en and fr');
```

Comment:
This line break is a bit weird. I'd prefer either having everything on one line, but since that may not be possible for length reasons I'd prefer one-line-per-parameter:

```js
assert.deepEqual(
  Object.keys(translations).sort(),
  ['fr', 'en'].sort(),
  'only returns en and fr'
);
```